### PR TITLE
Add validation tag 'required_in_project'

### PR DESCRIPTION
**Pull request by webrouse**
Povodny PR: https://github.com/keboola/keboola-as-code/pull/442

Tu rebasnute nad: https://github.com/keboola/keboola-as-code/pull/443

### CODE REVIEW

File: internal/pkg/validator/validator.go [hunk 1]
```diff
@@ -85,15 +93,52 @@ func newValidator() *wrapper {
...
 		}
+
+		// Register error message
 		v.registerErrorMessage(rule.Tag, rule.ErrorMessage)
 	}
 }
 
+func (v *wrapper) registerCustomRules() {
+	// Register default validation for "required_in_project"
+	// Some values are requited in the project scope, but ignored in the template scope.
```
Comment (JakubMatejka) [inline internal/pkg/validator/validator.go:117]: Tady jen malý typo v required 🙂

File: internal/pkg/validator/validator.go [hunk 2]
```diff
@@ -85,15 +93,52 @@ func newValidator() *wrapper {
...
+			return !fl.Field().IsZero()
+		},
+		ErrorMessage: "{0} is a required field",
+	})
+}
+
 // registerErrorMessage for a tag.
 func (v *wrapper) registerErrorMessage(tag, message string) {
+	if tag == "" {
+		panic(fmt.Errorf(`tag cannot by empty`))
```
Comment (JakubMatejka) [inline internal/pkg/validator/validator.go:136]: By -> be, myslím a pod tím taky

**Review by JakubMatejka — CHANGES_REQUESTED** [review]
Jen to typo

File: internal/pkg/validator/validator.go (hunk 2 above)
Comment (webrouse) [reply internal/pkg/validator/validator.go:136]: jj, dik fixnem

**Review by JakubMatejka — APPROVED** [review]

**Pull request merged**